src/index.html

```diff
@@ -1,14 +1,30 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Snake</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Snake</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Snake</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+    <div class="container">
+      <div class="block block__1">1</div>
+      <div class="block block__2">2</div>
+      <div class="block block__3">3</div>
+      <div class="block block__4">4</div>
+      <div class="block block__5">5</div>
+      <div class="block block__6">6</div>
```

Comment:
But you use nth-child, these classes don’t need

```suggestion
      <div class="block block--1">1</div>
      <div class="block block--2">2</div>
      <div class="block block--3">3</div>
      <div class="block block--4">4</div>
      <div class="block block--5">5</div>
      <div class="block block--6">6</div>
```